contracts/payment_splitter/Makefile.toml

```diff
@@ -0,0 +1,194 @@
+[env]
+friendbot = "https://friendbot-futurenet.stellar.org/"
+network = "futurenet"
+server = "https://horizon-futurenet.stellar.org/"
+
+[env.futurenet]
+friendbot = "https://friendbot-futurenet.stellar.org/"
+network = "futurenet"
+server = "https://horizon-futurenet.stellar.org/"
+
+[env.standalone]
+friendbot = "http://localhost:8000/friendbot"
+network = "standalone"
+server = "http://localhost:8000"
+
+[tasks.run_standalone_server]
+script = '''
+  docker run --rm -it \
+    -p 8000:8000 \
+    --name stellar \
+    stellar/quickstart:soroban-dev@sha256:a6b03cf6b0433c99f2f799b719f0faadbb79684b1b763e7674ba749fb0f648ee \
+    --standalone \
+    --enable-soroban-rpc
+'''
+
+[tasks.create_account_admin]
+script_runner = "@shell"
+script = '''
+  soroban config identity generate admin 2>/tmp/payment_splitter_error
+  ERR=`cat /tmp/payment_splitter_error`
+  if [ ! -z "$ERR" ]; then
+    echo "error=$ERR"
+    exit 1
+  fi
+  
+  curl "${friendbot}?addr=$(soroban config identity address admin)"
+'''
+
+[tasks.create_recipients]
+script = '''
+  for RECIPIENT in 1 2 3
+  do
+    soroban config identity generate recipient_$RECIPIENT 2>/tmp/payment_splitter_error
+    ERR=`cat /tmp/payment_splitter_error`
+    if [ ! -z "$ERR" ]; then
+      echo "$ERR"
+      exit 1
+    fi
+    
+    curl "${friendbot}?addr=$(soroban config identity address recipient_$RECIPIENT)"
+  done
+'''
+
+[tasks.accounts]
+dependencies = [
+  "create_account_admin",
+  "create_recipients"
+]
+
+[tasks.get_token]
+script = '''
+soroban lab token id --asset native --network ${network} > /tmp/token_id.txt
+'''
+
+[tasks.deploy]
+script = ''' 
+  soroban contract deploy \
+  --source admin \
+  --wasm ../../target/wasm32-unknown-unknown/release/payment_splitter.wasm \
+  --network ${network} > /tmp/contract_id.txt 2>/tmp/payment_splitter_error
+
+  ERR=`cat /tmp/payment_splitter_error`
+  if [ ! -z "$ERR" ]; then
+    echo "$ERR"
+    exit 1
+  fi
+'''
+
+[tasks.invoke_init]
+script = '''
+  soroban contract invoke \
+  --id $(cat /tmp/contract_id.txt) \
+  --source admin \
+  --network ${network} \
+  -- \
+  init \
+  --admin $(soroban config identity address admin) \
+  --token $(cat /tmp/token_id.txt) \
+  --stakeholders $(soroban config identity address recipient_1),$(soroban config identity address recipient_2),$(soroban config identity address recipient_3)  \
```

Comment:
```suggestion
  --stakeholders '[ "'$(soroban config identity address recipient_1)'","'$(soroban config identity address recipient_2)'","'$(soroban config identity address recipient_3)'" ]' \
```

Reply:
I was getting errors when executing this locally. Looks like this needs to be valid json even being just a simple list. Output of `--help`:

```bash
 --stakeholders <Array<Address>>
          Example:
            --stakeholders '[ "GDIY6AQQ75WMD4W46EYB7O6UYMHOCGQHLAQGQTKHDX4J2DYQCHVCR4W4" ]' --stakeholders <Array<Address>>
```